core/src/main/scala/org/apache/spark/ui/jobs/UIData.scala

```diff
@@ -63,7 +64,7 @@ private[jobs] object UIData {
     /* Stages */
     var numActiveStages: Int = 0,
     // This needs to be a set instead of a simple count to prevent double-counting of rerun stages:
-    var completedStageIndices: OpenHashSet[Int] = new OpenHashSet[Int](),
+    var completedStageIndices: mutable.HashSet[Int] = new mutable.HashSet[Int](),
```

Comment:
Did you disagree this? But the `OpenHashSet` doesn't support remove operation. Do you have any suggestion on this?